tests/test-sources/plugins/by-name/rainbow/default.nix

```
@@ -0,0 +1,145 @@
+{
+  empty = {
+    plugins.rainbow.enable = true;
+  };
+
+  default = {
```

```suggestion
  defaults = {
```

Minor nit, we use this naming convention.